helix-term/src/commands.rs

```diff
@@ -781,7 +781,7 @@ fn trim_selections(cx: &mut Context) {
         let idx = ranges
             .iter()
             .position(|range| range.overlaps(&primary))
-            .unwrap_or(ranges.len() - 1);
+            .unwrap_or_else(|| ranges.len() - 1);
```

Comment:
len() should be an O(1) lookup - is there an advantage to using a closure here?

Reply:
It was part of the first commit to resolve `cargo clippy` warning related to lazily evaluation of `saturating_sub` method. I will revert it.